File: Backend/Models/Shelter.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Backend.Models;

[Table("Shelters")]
public class Shelter : Person
{

    [MaxLength(60)]
    public string FantasyName { get; set; }

    public string? KeyPIX { get; set; }

    public string? About { get; set; }

    public List<Donation> Donations { get; set; } = new List<Donation>();


}

File: Frontend/src/types/Shelter.ts
import { DonationType } from './Donation'
import { ImageType } from './Image'
import { PersonType } from './Person'

export interface ShelterType extends PersonType {
  fantasyName: string
  images: ImageType[]
  about?: string | null
  keyPIX?: string
  donations: DonationType[]
}
